crates/oci-distribution/Cargo.toml

```diff
@@ -13,12 +13,11 @@ authors = [
 edition = "2018"
 
 [dependencies]
-reqwest = { version = "0.10.4", features = ["json", "stream"] }
+reqwest = { version = "0.10", features = ["json", "stream"] }
 anyhow = "1.0"
-tokio = {version  = "0.2.14", features = ["macros", "fs"] }
+tokio = {version  = "0.2", features = ["macros", "fs"] }
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 www-authenticate = "0.3.0"
-hyperx = "0.13"
-chrono = { version = "0.4", features = ["serde"] }
+hyperx = "0.13.0"
```

Comment:
Why does this one have a fully qualified version (with patch number), but the others don't?

Reply:
Whoops. Will fix